### ISSUE TITLE
field order

### ISSUE THREAD

**Issue by Ghastor** [RandomEngy/VidCoder#967]
### Feature details

Please, improve deinterlacing method swap frame order top/bottom mode.

Need to deinterlacing to 50 fps with bottom field first order.
Not problem with h264 using interlaced mode and bff=1 on empty box and 25 fps for output.

Thanks!

**Comment by RandomEngy** [comment]
What are you asking for, exactly? Improve the user interface? Or change the behavior with some specific encoding settings?

**Comment by Ghastor** [comment]
hi RandomEngy!
I'm asking for little menù near deinterlacing method to swap bottom/top field first for yadif or decomb mode.
at the moment only top field first works ok, but some pal dvd are with bff.
very bad output in 50 fps mode.
thanks

**Comment by RandomEngy** [comment]
I think you want to use `parity=1` in the Custom preset for the Decomb filter to get bottom field first.

**Comment by Ghastor** [comment]
never used custom mode at this time, never tried...

**Comment by RandomEngy** [comment]
If you tried it, you might be able to accomplish what you want to do.

**Comment by Ghastor** [comment]
how to use in custom format?

**Comment by RandomEngy** [comment]
Type `parity=1` in the box. If there are multiple settings they are separated by `:` . It gives you a list of valid keys when you hover over the text box.

**Comment by Ghastor** [comment]
video filter tab under custom or in codec setting box ?

**Comment by RandomEngy** [comment]
video filter tab

**Comment by Ghastor** [comment]
in video filter tab using decomb, BOB, custom, then  block-height=16: block-thresh=40: block-width=16: filter-mode=2: mode=3: motion-thresh=1: spatial-metric=2: spatial-thresh=1: parity=1 

framerate 50 fps under codec video and h265 10 bit main 10

my output is 25 fps ?!?

**Comment by RandomEngy** [comment]
25 FPS is standard for interlaced PAL, right?

**Comment by Ghastor** [comment]
yes, 25 fps is interlaced PAL. But I think i'm wrong for setting variable frame rate instead constant frame fate.
it was a carelessness.
Now my output is 50 fps and seems ok !
Thanks for your reply RandomEngy.
Setting with parity:1 seems ok, I've asking to you to add this simple menù TFF/BTT to make my work easier.
I've ask to you another enanchement later. Using LUT 3d by adobe to improve/change color.
